teacher.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Teacher < Person
+  def initialize(age, specialization, name = 'Unknown', parent_permission: true)
+    super(age, name, parent_permission)
```

Comment:
- [ ] As remarked by the previous reviewer, kindly change the `super` method call to the following, as you're using the colon notation for `parent_permission`.

```ruby
super(age, name, parent_permission: parent_permission)
```